docs/recipes/virtualenv.rst

```diff
@@ -0,0 +1,65 @@
+Setting Up a Python Virtualenv for SCons
+----------------------------------------
+
+It is often useful to set up a virtualenv when working with a project
+that uses SCons to build.  A virtualenv is a way to create an
+isolated execution environment - you can install whatever you need
+there, and change versions as needed, without affecting anything
+else on your system that uses Python.  Here is an example session
+for setting one up:
+
+.. code:: shell
+
+   $ cd Work
+   $ python -m venv myvenv
+   $ cd myvenv
```

Comment:
You shouldn't need to cd into myenv to source bin/activate

For windows you use scripts/activate.bat or activate.ps1(for powershell)